cmake/find_llvm.cmake

```diff
@@ -65,7 +65,7 @@ else()
   # Get LLVM version
   _run_llvm_config(LLVM_PACKAGE_VERSION "--version")
   # Try x.y.z patern
-  set(_llvm_version_regex "^([0-9]+)\\.([0-9]+)\\.([0-9]+)$")
+  set(_llvm_version_regex "^([0-9]+)\\.([0-9]+)\\.([0-9]+)(svn)?$")
```

Comment:
Hmm looking at LLVM 2.9's code (damn it's old... `llvm-config` is a perl script in that version...) I don't see any evidence that `svn` will be appended so I'm going to accept this.